data/frontier jobs.txt

```diff
@@ -0,0 +1,156 @@
+			choice
+				`(Join their game.)`
+					goto gamble
+				`(Ignore them and leave quickly.)`
+					defer
+
+			label gamble
+			`	You saunter confidently over to the table. A closer examination all but confirms your suspicion that they're pirates. Not one of them looks older than 19, and they all sport fearsome tattoos and dangerous-looking sidearms. You put on a brave face and they deal you into their card game. After a few rounds it becomes abundantly obvious that they're cheating, and making little effort to hide this from you.`
+			branch "good luck" "bad luck"
+				random < 30
+
+			label "good luck"
+			`	Fortunately, you're able to outsmart them anyway and win the game, winding up with a sizable pile of credit chips. They glower but do not molest you as you quickly seek the safety of a public place. You thank your lucky stars: you're nearly 25,000 credits richer and still in one piece after a face-to-face encounter with pirates.`
+				accept
+
+			label "bad luck"
+			`	You're starting to bleed money, with more than 10,000 credits in losses already.`
+			choice
+				`(Call them out on their cheating.)`
+					goto call
+				`(Say nothing and continue playing.)`
+					goto lose
+
+			label lose
+			`	You grit your teeth and lose hand after hand before the game finally, mercifully ends. Discretion may be the better part of valor, but it's lost you almost 15,000 credits. The pirates laugh at you and gloat over their "winnings" as you slink away, your pride stung.`
+				decline
+
+			label call
+			`	This is intolerable. You start to raise a stink about their blatant cheating.`
+			`	"We're just playin' a game here, see? You'd better shut your big mouth if you know what's good for you," the meanest-looking one growls at you. "We can make this ugly real fast."`
+			`	The four thugs stare at you menacingly and start to reach for their guns.`
+			choice
+				`(Shoot first!)`
+					goto shoot
+				`(Grab the pot and run!)`
+					goto run
+				`(Let them fleece you.)`
+					goto lose
+
+			label shoot
+			branch "shoot and die" "shoot and run"
+				random < 30
+
+			label "shoot and die"
+			`	Unfortunately they're faster than you. Your mutilated corpse hits the floor with a wet thump.`
+				die
+
+			label "shoot and run"
+			`	Grateful that you never leave your ship unarmed, your hand flies toward the pistol at your thigh. In one smooth motion you draw the weapon and fire at the pirate who just threatened you. The bolt punches a fist-sized hole in the center of his chest, and he collapses like a sack of potatoes. His shocked comrades fumble in horror for their weapons but you've taken them by surprise and quickly gun down another before deciding it's time to escape. You grab some of the chits in the pile of credits on the way out!`
```

Comment:
Remove `in one smooth motion` so it's just `You draw the weapon...`
`bolt` -> `shot`
Remove `fist-sized`
`he collapses to the ground` instead of `like a sack of potatoes`.
`for their weapons, but` (include comma)
`chits` -> `chips`?